3/resolvers.js

```diff
@@ -0,0 +1,89 @@
+const uid = require('uuid/v4');
+const assignees = require('./datasource.js').assignees;
+const todos = require('./datasource.js').todos;
+const jwt = require('jsonwebtoken');
+
+const resolvers = {
+    Query: {
+      assignees: (parent, {token}) => { 
+        var decode = jwt.verify(token, 'shhhhh')
+        if(decode.auth == 'der richtige inhalt') {
+          return assignees;
+        } else {
+          return null;
+        }
+      },
+      assignee: (parent, {id}) => { return assignees.filter(x => x.id = id); },
+      todos: () => { return todos; },
+      todo: (parent, {id} ) => { return todos.filter(x => x.id === id)[0]; },
+      todosForAssigneeId: (parent, {id}) => { return todos.filter(x => x.assigneeId === id); },
+      countTodos: () => { return todos != null? todos.length : 0; },
+      countAssignees: () => { return assignees != null? assignees.length : 0; },
+      todosByState: (parent, {state}) => { return todos.filter(x => x.state === state); }
```

Comment:
I am not sure, but I think it is nor neccessary to implement individual queries for these functionalities. 

![grafik](https://user-images.githubusercontent.com/33908029/70067569-bb7aac00-15ee-11ea-8cd7-9ab43663087d.png)
You could just use the GraphQL filter function